noxfile.py

```diff
@@ -114,7 +115,7 @@ def run_tests(session):
 @nox.session(name="docs", reuse_venv=True)
 def build_docs(session):
     envbindir = session.bin
-    session.install("-e", ".[docs]")
+    session.install("-e", ".[all,docs]")
```

Comment:
Should this be just `docs`?

Reply:
These are also requiring other dependencies as well, i remember pybtex, tqdm and etc. so I ended up installing [all] too.
Previously it was `[dev,plot,cite,latexify,bpx,tqdm]`